modules/nodes-nodes-working-deleting-bare-metal.adoc

```diff
@@ -24,21 +24,25 @@ the following steps:
 $ oc adm cordon <node_name>
 ----
 
-. Drain all pods on your node:
+. Drain all pods on the node:
 +
 [source,terminal]
 ----
 $ oc adm drain <node_name> --force=true
 ----
 
-. Delete your node from the cluster:
+.. If the node is unresponsive, it might be running a workload that writes to shared storage. To avoid data corruption, power down the physical hardware.
```

Comment:
This kind of looks odd that there is a random "a" substep. Just checking whether that was intentional, or if it was either supposed to be a main step, or just a sentence after step 2.

Looking at the addition of step 4, I'm thinking that it was meant as a main step.